collections/Laud_Misc/MS_Laud_Misc_268.xml

```diff
@@ -52,63 +80,260 @@
                <msPart xml:id="MS_Laud_Misc_268-part1">
                   <msIdentifier>
                      <altIdentifier type="partial">
-                        <idno type="part">MS. Laud Misc. 268 – Part 1</idno>
+                        <idno type="part">MS. Laud Misc. 268 – Part A (fols. 1-42)</idno>
                      </altIdentifier>
                   </msIdentifier>
-                  <msContents>
+                  <msContents> <textLang mainLang="la">Latin</textLang>
                      <msItem n="1" xml:id="MS_Laud_Misc_268-part1-item1">
-                        <author key="person_61539765">Bede</author>
-                        <title key="work_950">Commentary on Acts</title>
-                        <bibl type="commentedOn">
-                           <title>Bible, Acts</title>
-                        </bibl>
-                        <note>(partly conflated with his Retractions, as in MS. Laud Misc. 312)</note>
-                        <textLang mainLang="la">Latin</textLang>
+                        <locus>(fols. 1ra-41rb)</locus>
+                        <rubric><note type="editorial">[running head]</note> Actus ap<ex>osto</ex>l<ex>orum</ex></rubric>
+                        <rubric>Incipit prefacio Bede presbiteri in actus apostolorum</rubric>
+                        <incipit>Domino in Christo desiderantissimo</incipit>
+                        <rubric><locus>(fol. 1vb)</locus> Incipit <del rend="erasure">textus</del> eiusdem</rubric>
```

Comment:
I think it would be clearer to use extra `<msItem>`s to wrap the additional incipit/explicit pairs.